App/Server/Dockerfile.e2e

```diff
@@ -0,0 +1,27 @@
+# Use the official .NET 9 SDK image
```

Comment:
Let's not user a separate Dockerfile just for this. There is already an existing Dockerfile that we should re-use. I think you created this one because I mentioned a specific port, but forget about that and just use the same strategy as the Test/Postman docker-compose.yml